console/war/src/main/resources/application.properties

```diff
@@ -40,14 +44,9 @@ csrf.enabled=true
 csrf.cookie.path=
 
 ## Enables authentication for the Frank!Console
-#application.security.console.authentication.type=IN_MEMORY
-#application.security.console.authentication.username=Admin
-#application.security.console.authentication.password=Nimda
-
-## Enables authentication for the Ladybug (if present)
-#application.security.testtool.authentication.type=IN_MEMORY
-#application.security.testtool.authentication.username=Admin
-#application.security.testtool.authentication.password=Nimda
+application.security.console.authentication.type=IN_MEMORY
+application.security.console.authentication.username=Admin
+application.security.console.authentication.password=Nimda
```

Comment:
Doublecheck: Weet je zeker dat je die nu altijd enabled wil hebben?
Of is dit per ongeluk meegegaan?

Reply:
OEI!